### PR TITLE
Updated README, added routes, fixed PUT warning

### PR DESCRIPTION
**Pull request by miesoz**

### CODE REVIEW

**Review by kungoyal — DISMISSED** [review]

File: goon-universe/app.js [hunk 1]
```diff
@@ -152,6 +152,7 @@ router.route('/scenarios/initialReflection')
         scenarioID = req.body.scenarioID
         studentID = req.body.studentID
         data = req.body.data
+        let flag = true
```
Comment (rlee287) [inline goon-universe/app.js]: Could this be renamed to something more descriptive?

File: goon-universe/app.js [hunk 2]
```diff
@@ -514,6 +585,7 @@ router.route('/scenarios/middleReflection')
...
```
Comment (rlee287) [inline goon-universe/app.js]: Should be renamed to something more descriptive

File: goon-universe/app.js [hunk 3]
```diff
@@ -603,6 +684,7 @@ router.route('/scenarios/finalReflection')
...
```
Comment (rlee287) [inline goon-universe/app.js]: Rename to something more descriptive

File: goon-universe/app.js (hunk 1 above)
Comment (kungoyal) [reply goon-universe/app.js]: Done

**Review by rlee287 — DISMISSED** [review]
Checks for whether the DB query succeeded or not may need changes later, but the rest looks good

**Review by kungoyal — APPROVED** [review]

**Review by rasdy — COMMENTED** [review]
Looks fine to me, but I'll need Anastasia to assess the action endpoints.

**Comment by agalperina** [comment]
Ive tried out the action endpoints looks good to me as well!

**Review by agalperina — APPROVED** [review]

**Pull request merged**